apps/src/templates/manageStudents/ManageStudentsTable.jsx

```diff
@@ -729,6 +747,31 @@ class ManageStudentsTable extends Component {
               )}
             </div>
           )}
+          {noSectionCode.includes(loginType) && (
+            <div style={styles.sectionCodeBox}>
+              {i18n.sectionCodeWithColon()}
+              <strong>
+                {` ${i18n.notApplicable()} `}
+                <a onClick={() => this.showSectionCodeDialog()}>
+                  {i18n.whyWithQuestionMark()}
+                </a>
+              </strong>
+            </div>
+          )}
+          <div>
```

Comment:
(For my own learning) Why put this div outside the `noSectionCode.includes...` conditional? Does the dialog run into issues if it's conditionally rendered?

Reply:
There’s no downside to conditionally rendering. It should have been there to begin with – that’s my bad!

Reply:
👍